cypress/e2e/bulk-edit/permissions/bulk-edit-csv-permissions.cy.js

```diff
@@ -42,9 +41,7 @@ describe('bulk-edit', () => {
       cy.login(userWithCsvPermissions.username, userWithCsvPermissions.password);
```

Comment:
we can't automate this test-case C353537 because of verifying the visual clue. we need to move it to manual type . we need to discuss it and possibly move to extended path type after confirm this changes.